Components/common/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import styles from './styles';
+import { TouchableHighlight } from 'react-native-gesture-handler';
+
+const Button = props => {
+	return (
+		<View style={styles.container}>
+			<TouchableHighlight {...props} style={{ ...styles.button, ...props.style }}>
+				<Text style={styles.buttonText}>{props.children}</Text>
```

Comment:
@kalex19 

This is interesting that you separated the button into its own component. Will this component be reused for all the buttons of the app?

Reply:
Yeah, it decreases the amount of code in each component and keeps the button styles the same across the app.